src/2_sort_words.js

```diff
@@ -1,3 +1,22 @@
-module.exports = function sortWords() {
-  // TODO Implement me.
-};
+/*module.exports = */ 
+
+function sortWords (arr) {
+  let outputObj = 
+      {
+    even: [],
+    odd: []
+      };
+  for (let i=0; i< arr.length; i++) {
+    arr[i].length % 2 == 0 ? outputObj.even.push(arr[i]) : outputObj.odd.push(arr[i]);
```

Comment:
trenary op. side effect: https://stackoverflow.com/questions/55349332/using-ternary-operator-just-for-the-side-effect-good-or-bad